continuous_integration/gpuci/environment-3.8.yaml

```diff
@@ -37,7 +37,7 @@ dependencies:
 - sqlalchemy<2
 - tpot
 - tzlocal=2.1
-- uvicorn=0.13.4
+- uvicorn>=0.13.4
```

Comment:
Mirrors a change made to the images in https://github.com/rapidsai/dask-build-environment/pull/64 that we need to make in order to pull in the latest `cuml` nightlies (the lack of which is causing the current GPU failures)